caddy/caddy.go

```diff
@@ -322,17 +292,6 @@ func Wait() {
 // are no other errors, this function always returns at least the
 // default Caddyfile.
```

Comment:
We should have this godoc comment updated too, removing the part about piping in as part of a restart.